docs/release-notes/8.5.asciidoc

```diff
@@ -1,6 +1,22 @@
 [[release-notes-header-8.5.0]]
 == 8.5
 
+[discrete]
+[[release-notes-8.5.1]]
+=== 8.5.1
+
+[discrete]
+[[bug-fixes-8.5.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that caused {elastic-endpoint}s (version 8.5.0) running on Linux systems with many CPUs to sometimes become unhealthy https://github.com/elastic/endpoint/issues/34[#34].
```

Comment:
```suggestion
* Fixes a bug that caused {elastic-endpoint}s (version 8.5.0) running on Linux systems with many CPUs to sometimes become unhealthy (https://github.com/elastic/endpoint/issues/34[#34]).
```

Reply:
Note to self to confirm next week that this has been fixed.

Reply:
In addition to adding the parentheses around the issue #, we could probably omit the version #. We don't usually mention the version # where a bug was introduced, just that it's been fixed.
```suggestion
* Fixes a bug that caused {elastic-endpoint}s running on Linux systems with many CPUs to sometimes become unhealthy (https://github.com/elastic/endpoint/issues/34[#34]).
```